src/cmd/tools/annotation_checker/main/README.md

```diff
@@ -2,26 +2,30 @@
 
 `annotation_checker` is a utility that reads TSDB file sets and analyzes timeseries for missing initial annotations or annotation rewrites.
 
-By default, this tools output the list of series IDs that have no annotation with the first datapoint in the block. If `--annotation-rewritten` flag is used, the output will contain series IDs of metrics with rewritten annotations instead.
+By default, this tools output the list of series IDs that have no annotation with the first datapoint in the block. If `--annotation-filter` is set to `annotation-rewritten`, the output will contain series IDs of metrics with rewritten annotations instead.
 
 # Usage
 ```
 $ make annotation_checker
 $ ./bin/annotation_checker
-Usage: annotation_checker [-R] [-b value] [-n value] [-p value] [-s value] [-t value] [-v value] [parameters ...]
- -b, --block-start=value
-                    Block Start Time [in nsec]
- -n, --namespace=value
-                    Namespace [e.g. metrics]
- -p, --path-prefix=value
-                    Path prefix [e.g. /var/lib/m3db]
- -R, --annotation-rewritten
-                    Filters metrics with annotation rewrites
- -s, --shard=value  Shard number, or -1 for all shards in the directory
- -t, --fileset-type=value
-                    flush|snapshot
- -v, --volume=value
-                    Volume number
+Usage: annotation_checker [-P] [-a no-filtering|no-initial-annotation|annotation-rewritten] [-b nsec] [-f string] [-n string] [-p string] [-s int] [-t flush|snapshot] [-v int] [parameters ...]
+ -a, --annotation-filter=no-filtering|no-initial-annotation|annotation-rewritten
```

Comment:
Nit: I suggest shortening things a little, e.g. ` --annotation-filter=no-filtering` -> ` --annotation-filter=none`,
also in `annotation_checker --annotation-filter=annotation-rewritten` we repeat `annotation` all over the place, perhaps could use `--annotation-filter=rewritten`.